src/app/ApplicationTemplate.Shared.Views/App.xaml.cs

```diff
@@ -33,23 +32,11 @@ public App()
 
 	public Window CurrentWindow { get; private set; }
 
-	protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
-	{
-		InitializeAndStart();
-	}
-
-	/*
-	protected override void OnActivated(IActivatedEventArgs args)
```

Comment:
Don't we need to keep this when launching from custom schemes, universal links, or Android App Links?

Reply:
Doesn't exist on WinUI.

Reply:
https://github.com/microsoft/microsoft-ui-xaml/issues/3835#issuecomment-868820683

Reply:
I think it does contain arguments.
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.launchactivatedeventargs?view=windows-app-sdk-1.2

Reply:
Actually, you should create an Uno discussion about this now that the `OnActivated` doesn't exist? anymore on WinUI. What's the entry point when launching iOS and Android from urls and push notifications?

Reply:
https://github.com/unoplatform/uno/discussions/10803

Reply:
We'll need to follow this discussion for update and see in future Pull Request.